staging/com.scoreSmart/device.js

```diff
@@ -3,75 +3,65 @@
 const Tp = require('thingpedia');
 const unirest = require("unirest");
 
-module.exports = class ScoreDevice extends Tp.BaseDevice {
+module.exports = class GoalDevice extends Tp.BaseDevice {
   constructor(engine, state) {
     super(engine, state);
-    this.uniqueId = 'com.scoreSmart';
-    this.name = 'ScoreSmart';
+    this.uniqueId = 'com.goal_smart';
+    this.name = 'GoalSmart';
```

Comment:
Is this a name you're making up yourself or is it an actual service that exists?

Reply:
This is a name that I have made up myself.